changelog/add-5406-remove-payment-button-on-0-cart

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: fix
+
+Remove Google Pay and Apple Pay for zero balance owed carts.
```

Comment:
I think we can make this more clear. How about:

```suggestion
Hide Google Pay and Apple Pay buttons when total amount is zero on item details, cart, and checkout pages
```

Reply:
fixed in [b2e4c9b](https://github.com/Automattic/woocommerce-payments/pull/6334/commits/b2e4c9b0b25311c882f56a142b867916a64d6880) . thx .